validator/test/fixtures/gke_enable_stackdriver_monitoring/constraints/enable_gke_stackdriver_monitoring/data.yaml

```diff
@@ -18,7 +18,8 @@ metadata:
   name: enable_gke_stackdriver_monitoring
   annotations:
     description: Ensure stackdriver monitoring is enabled on a GKE cluster
-    benchmark: CIS11_7.02
```

Comment:
IMO we should drop the bundle info from test data entirely.

Reply:
@morgante Can you clarify what you mean here?

Reply:
For the constraints in the `tests` directly we can simply remove the `benchmark` annotation entirely instead of replacing it with the new format.

Reply:
ah, gotcha. @joecheuk had asked me to add this in #132. @joecheuk thoughts?

Reply:
I mightve commented on the wrong file, as long as they are in the constraint files under samples/ then it's fine.